From 06a7c5dd5bd47b8f4bb919359bf6b772dfb98523 Mon Sep 17 00:00:00 2001
From: Dan Kephart <dan.kephart@lairdtech.com>
Date: Tue, 15 Jul 2014 12:38:56 -0400
Subject: [PATCH] Making cfg80211 a module to easily support Linux backports
 testing

Our Linux backports driver requires cfg80211 to be a module.  With this
change, the backports driver and modules can be dropped on the WB and
tested easily.

BUG 5835
---
 arch/arm/configs/wb45n_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/configs/wb45n_defconfig b/arch/arm/configs/wb45n_defconfig
index 25525b2..d4e8359 100644
--- a/arch/arm/configs/wb45n_defconfig
+++ b/arch/arm/configs/wb45n_defconfig
@@ -128,7 +128,7 @@ CONFIG_BT_BNEP_MC_FILTER=y
 CONFIG_BT_BNEP_PROTO_FILTER=y
 CONFIG_BT_HIDP=m
 CONFIG_BT_HCIBTUSB=m
-CONFIG_CFG80211=y
+CONFIG_CFG80211=m
 CONFIG_NL80211_TESTMODE=y
 CONFIG_CFG80211_DEVELOPER_WARNINGS=y
 CONFIG_CFG80211_CERTIFICATION_ONUS=y
-- 
1.8.3.2

